src/com/company/reverse/Reverse.java

```diff
@@ -0,0 +1,21 @@
+package com.company.reverse;
+
+public class Reverse {
+    public static void main(String[] args) {
+
+
+        System.out.println(reverse("ad12fgiu57"));
+    }
+
+    private static String reverse(String s) {
+        StringBuilder result = new StringBuilder(s.replaceAll("[\\d+]", "")).reverse();
```

Comment:
Please give it more meaningful name instead of generic `result`

Reply:
Done